app/src/main/java/edu/agh/susgame/front/model/graph/GameGraph.kt

```diff
@@ -5,31 +5,45 @@ import edu.agh.susgame.front.model.PlayerId
 import edu.agh.susgame.front.util.Coordinates
 
 class GameGraph(
-    val nodes: MutableMap<NodeId, Node>,
-    val edges: MutableMap<EdgeId, Edge>,
+    val nodes: Map<NodeId, Node>,
+    val edges: Map<EdgeId, Edge>,
+    val players: Map<PlayerId, Player>,
     val paths: MutableMap<PlayerId, Path>,
```

Comment:
Wydaje mi się, czy ta zmienna jest zawsze pustą mapą? 
Gdy klikam `crtl+b` na nazwie, to jedyne co widzę, to inicjalizację pustą mapą i próbę odczytu jakiegoś `PlayerId`

Czy w takim razie klasa `Path` jest w ogóle używana? Ta mapa to chyba jej jedyne użycie

Reply:
Docelowo będzie to zmienna przysyłana od serwera, z której będzie trzeba brać informacje o ścieżkach wszystkich graczy. Na razie de facto informacja o trasach jest trzymana tylko w krawędziach

Reply:
Spoko spoko, myślałem że coś powinno tu być